backend/accounts/views.py

```diff
@@ -39,6 +40,34 @@
 from accounts.verification import sendEmailVerification, sendSMSVerification
 
 
+def image_upload_helper(request, user, keyword, field):
```

Comment:
We should probably be validating the size of the uploaded image at any point in the code (doesn't have to be here necessarily, just making the comment here)? Want to make sure users cannot upload arbitrarily sized profile pictures.